src/aiida_wannier90_workflows/workflows/optimize.py

```diff
@@ -299,9 +311,16 @@ def get_builder_from_protocol(  # pylint: disable=arguments-differ
         builder = cls.get_builder()
         builder._data = parent_builder._data  # pylint: disable=protected-access
 
+        # Run plotting in a separate step if wannier_plot is True since plotting Wannier dunctions is heavy
+        if (
+            builder["wannier90"]["wannier90"]["parameters"]
+            .get_dict()
+            .get("wannier_plot", False)
+        ):
+            builder.separate_plotting = True
+
```

Comment:
Maybe move these lines after the next conditional of `if reference_bands:`, so that we only `separate_plotting` if we are going to `optimize_disproj`
```
if builder.optimize_disproj == True:
    YOUR_LINES_HERE
```

Reply:
Thanks! Do we still want to automatically separate plotting only if `wannier_plot == True` or always if `optimize_disproj == True`? The latter will cause a warning in case no plotting inputs are specified.

Reply:
I mean we auto set `separate_plotting = True` only if both `wannier_plot = True` and `optimize_disproj = True`